examples/client/client.c

```diff
@@ -182,6 +188,17 @@ static int ClientBenchmarkConnections(WOLFSSL_CTX* ctx, char* host, word16 port,
             if (ssl == NULL)
                 err_sys("unable to get SSL object");
 
+    #ifdef WOLFSSL_TLS13
+            if (!benchResume || !noPskDheKe) {
+                if (wolfSSL_UseKeyShare(ssl, WOLFSSL_ECC_SECP256R1)
```

Comment:
Let's pick a default key share like p256, have that be the default, and then allow the user to override by calling this function.  That eliminates the need for every user to call every time.